packages/core/demo/data/bubble.ts

```diff
@@ -31,6 +31,14 @@ export const bubbleDoubleLinearOptions = {
 		radiusMapsTo: 'surplus',
 		radiusLabel: 'Surplus',
 	},
+	legend: {
+		additionalItems: [
+			{
+				type: 'radius',
+				name: 'Surplus',
+			}
+		]
+	}
```

Comment:
this is redundant, the demo works the same way without it

Reply:
This will override the default radius label item with the `name: Surplus`.

Reply:
👍🏻